### ISSUE TITLE
Changing IP ban message

### ISSUE THREAD

**Issue by xrampagex** [BigBrotherBot/big-brother-bot#402]
Is there anyway to edit the IP ban message that shows when a client is IP banned?

I think the IP address showing in global chat is a little too invasive.

I have tried a few different ways to no avail.

**Comment by BestNoob** [comment]
Yep i thought the same exact location and IP shouldnt be shown public.
I do not remember the file to edit but you can hide it by setting this only displayed for admins with the permission system.

**Comment by xrampagex** [comment]
We edited the init__.py and took out the IP address from being shown.
